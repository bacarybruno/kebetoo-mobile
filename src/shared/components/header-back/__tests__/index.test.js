import setupTest from '@app/config/jest-setup'

import HeaderBack from '../index'

const givenHeaderBack = setupTest(HeaderBack)({
  tintColor: 'red',
})
const givenHeaderClose = setupTest(HeaderBack.Close)({
  tintColor: 'red',
})

it('renders HeaderBack', () => {
  const { wrapper } = givenHeaderBack()
  expect(wrapper.toJSON()).toMatchSnapshot()
})

it('renders HeaderClose', () => {
  const { wrapper } = givenHeaderClose()
  expect(wrapper.toJSON()).toMatchSnapshot()
})
