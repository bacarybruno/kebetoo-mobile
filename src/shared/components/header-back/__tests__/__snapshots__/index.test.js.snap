// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders HeaderBack 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "marginLeft": 16,
        "width": 24,
      },
      undefined,
    ]
  }
>
  <Text
    allowFontScaling={false}
    style={
      Object {
        "color": "red",
        "fontFamily": "icomoon",
        "fontSize": 20,
        "fontStyle": "normal",
        "fontWeight": "normal",
      }
    }
    text="kebeticon-chevron-left"
  />
</View>
`;

exports[`renders HeaderClose 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "marginLeft": 16,
        "width": 24,
      },
      undefined,
    ]
  }
>
  <Text
    allowFontScaling={false}
    style={
      Object {
        "color": "red",
        "fontFamily": "Ionicons",
        "fontSize": 40,
        "fontStyle": "normal",
        "fontWeight": "normal",
      }
    }
    text="ionicon-ios-close"
  />
</View>
`;
